packages/components/pie-modal/src/modal.scss

```diff
@@ -136,12 +136,8 @@
         @media (min-width: $breakpoint-wide) {
             --modal-footer-padding: var(--dt-spacing-e);
         }
-    }
 
-    // When hasStackedActions is set
-    // change the direction of the footer flex container so buttons are full width of modal
-    &[hasstackedactions] {
-        & .c-modal-footer {
+        &.c-modal-footer--stackedActions {
```

Comment:
This stacked actions class is now applied to the footer in `renderModalFooter`. I thought this was a bit nicer than adding the class to the modal and then applying these styles to the footer when it is inside a modal with that class.